src/current/_data/redirects.yml

```diff
@@ -270,6 +268,18 @@
   sources: ['import-data.md']
   versions: ['v2.1', 'v19.1', 'v19.2', 'v20.1', 'v20.2', 'v21.1']
 
+- destination: molt/molt-fetch.md
```

Comment:
I couldn't find any other examples of a directory in the `destination` field except for a handful below with `stable` in them. When I try to get to a page that should redirect to `molt/molt-fetch` in your PR, it isn't redirecting and it's putting an extra `molt/` in the source, so redirecting `v23.1/molt-fetch` to `v231/molt/molt-fetch` instead of to `/molt/molt-fetch`. I wonder if others with more experience of redirects can help.